src/ray/gcs/gcs_server/gcs_worker_manager.h

```diff
@@ -73,6 +85,16 @@ class GcsWorkerManager : public rpc::WorkerInfoHandler {
   std::vector<std::function<void(std::shared_ptr<WorkerTableData>)>>
       worker_dead_listeners_;
 
+  /// A list where workers are store as pairs of (WorkerID, Timestamp).
+  /// The workers are sorted according to the timestamp, and the oldest is at the head of the list.
+  /// @note The pair consists of:
+  ///   - first: WorkerID (identifier for the worker)
+  ///   - second: Timestamp (time when the worker was last updated or added)
+  std::list<std::pair<WorkerID, int64_t>> sorted_dead_worker_list_;
+
+  /// Max number of dead workers allowed in the storage.
+  const size_t max_num_dead_workers_ = 1000;
```

Comment:
No default value needed here since we always use the config value.

Reply:
ok.